src/registrar/models/utility/orm_helper.py

```diff
@@ -0,0 +1,8 @@
+from django.db.models.expressions import Func
+
+
+class ArrayRemove(Func):
```

Comment:
Suggestion:  rename to ArrayRemoveNull

On first read the name implied removing an array, the comment clarified this. (also hope you don't mind me peaking early on this draft)